var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -35,6 +35,7 @@ class Seacas(CMakePackage):
 
     # ###################### Versions ##########################
     version("master", branch="master")
+    version("2024-06-27", sha256="aeee5aa11ca52eba884d4fe7a9cd994d1c4b7c27")
```

Comment:
Is this the correct sha256?  It looks too short...

Reply:
I agree. I hit the "Copy full SHA" button next to the tag and that was what it gave me. I tried it a couple times because I was confused as well. Would you happen to remember how you got the longer SHA for the last ones? I will happily update it.

Reply:
I use ` spack checksum seacas` and then you can pick how many versions to checksum...

Reply:
I get:
```
version("2024-06-27", sha256="a28db6aa3d03ff0a54a091210cf867661427f0b22ac08f89a4cc3bd8e0c704b2")
```

Reply:
Yup, I am updating now. I think the difference is the SHA for the commit vs the SHA for the .tar file.